containers/local.env

```diff
@@ -8,6 +8,8 @@ FILES_DB_BUCKET=testbucket
 FILES_DB_ACCESS_KEY=minioadmin
 FILES_DB_SECRET_KEY=minioadmin
 
+DURATION_BETWEEN_METRICS_REFRESH=5000
```

Comment:
Instead of DURATION, you should specify the unit of 5000.

Reply:
I don't think `MILLIS` is the right unit to use, I think this should be measured in terms of minutes, and I think 5 minutes is probably the right value for us.